src/lib/widgets/now-playing/lib/detect-swipe.ts

```diff
@@ -0,0 +1,66 @@
+const PIXEL_THRESHOLD = 60;
+
+type SwipeCallbacks = Partial<Record<'left' | 'right' | 'up' | 'down', () => void>>;
+
+export const detectSwipe = (callbacks: SwipeCallbacks) => {
```

Comment:
suggestion (non-blocking): would be even cooler to make this a Svelte action, something like this:
```svelte
<div use:swipe={{ down: () => dispatch('minimize') }} />
```